From 9cc2c06d89d357a22065913abb0ab3ccda5a732b Mon Sep 17 00:00:00 2001
From: Mario Ceresa <mario.ceresa@upf.edu>
Date: Mon, 15 Dec 2014 12:22:00 +0100
Subject: [PATCH] Find and include CharLS

---
 CMake/3rdparty.cmake | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/CMake/3rdparty.cmake b/CMake/3rdparty.cmake
index 9fcca77..2ccda35 100755
--- a/CMake/3rdparty.cmake
+++ b/CMake/3rdparty.cmake
@@ -357,6 +357,20 @@
     endif()
   endif()
 
+  set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake)
+  
+  if(DCMTK_WITH_CHARLS)
+    find_package(CharLS)
+    include_directories(${CHARLS_INCLUDE_DIRS})
+    set(CHARLS_LIBS ${CHARLS_LIBRARIES})
+    if (NOT CHARLS_LIBS)
+      message(STATUS "Warning: CharLS not found. Cannot build!")
+    else(NOT CHARLS_LIBS)
+      message(STATUS "Info: DCMTK CHARLS support will be enabled")
+      set(WITH_CHARLS 1)
+    endif(NOT CHARLS_LIBS)
+  endif(DCMTK_WITH_CHARLS)
+
 endif(WIN32 AND NOT MINGW)
 
 if(NOT DEFINED DCMTK_WITH_STDLIBC_ICONV)
