From 6996b93b9ea3721ae970e4984d617fe050cd3b02 Mon Sep 17 00:00:00 2001
From: Mario Ceresa <mario.ceresa@upf.edu>
Date: Mon, 15 Dec 2014 12:00:53 +0100
Subject: [PATCH] Removed reference to bundled libcharls

---
 dcmjpls/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dcmjpls/CMakeLists.txt b/dcmjpls/CMakeLists.txt
index e114663..c92ac5b 100644
--- a/dcmjpls/CMakeLists.txt
+++ b/dcmjpls/CMakeLists.txt
@@ -2,6 +2,6 @@
 project(dcmjpls)
 
 # recurse into subdirectories
-foreach(SUBDIR libsrc libcharls apps include)
+foreach(SUBDIR libsrc apps include)
   add_subdirectory(${SUBDIR})
 endforeach()
