--- dcmtk-3.6.5/CMake/GenerateDCMTKConfigure.cmake.orig	2019-12-17 21:15:50.393593359 +0100
+++ dcmtk-3.6.5/CMake/GenerateDCMTKConfigure.cmake	2019-12-17 21:17:34.619695384 +0100
@@ -164,7 +164,7 @@
     set(DCM_DICT_DEFAULT_PATH "")
   endif()
   # Set default directory for configuration and support data.
-  set(DCMTK_DEFAULT_CONFIGURATION_DIR "${DCMTK_PREFIX}/${CMAKE_INSTALL_SYSCONFDIR}/dcmtk/")
+  set(DCMTK_DEFAULT_CONFIGURATION_DIR "${CMAKE_INSTALL_FULL_SYSCONFDIR}/dcmtk/")
   set(DCMTK_DEFAULT_SUPPORT_DATA_DIR "${DCMTK_PREFIX}/${CMAKE_INSTALL_DATADIR}/dcmtk/")
 endif()
 
