--- dcmtk-3.6.0/configure.orig	2009-09-04 11:08:47.000000000 +0200
+++ dcmtk-3.6.0/configure	2012-02-05 17:03:52.671202414 +0100
@@ -27,6 +27,6 @@
 else
 	for module in $modules ; do
 		echo "Running configure for module $module."
-		(cd $module && ./configure $* )
+		(set -x && cd $module && ./configure "$@" )
 	done
 fi
--- dcmtk-3.6.0/dcmdata/configure.orig	2010-06-04 11:52:19.000000000 +0200
+++ dcmtk-3.6.0/dcmdata/configure	2012-02-05 18:03:20.031321863 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmimage/configure.orig	2010-06-04 11:52:19.000000000 +0200
+++ dcmtk-3.6.0/dcmimage/configure	2012-02-05 18:04:34.167991012 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmimgle/configure.orig	2010-06-04 11:52:19.000000000 +0200
+++ dcmtk-3.6.0/dcmimgle/configure	2012-02-05 18:05:11.437992259 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmjpeg/configure.orig	2010-06-04 11:52:19.000000000 +0200
+++ dcmtk-3.6.0/dcmjpeg/configure	2012-02-05 18:05:44.074660019 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmjpls/configure.orig	2010-06-04 11:52:19.000000000 +0200
+++ dcmtk-3.6.0/dcmjpls/configure	2012-02-05 19:02:37.461440978 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmnet/configure.orig	2010-06-04 11:52:19.000000000 +0200
+++ dcmtk-3.6.0/dcmnet/configure	2012-02-05 19:04:15.808110939 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmpstat/configure.orig	2010-06-04 11:52:20.000000000 +0200
+++ dcmtk-3.6.0/dcmpstat/configure	2012-02-05 19:05:02.091445823 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmqrdb/configure.orig	2010-06-04 11:52:20.000000000 +0200
+++ dcmtk-3.6.0/dcmqrdb/configure	2012-02-05 19:05:46.411447305 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmsign/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/dcmsign/configure	2012-02-05 19:06:23.611448552 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmsr/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/dcmsr/configure	2012-02-05 19:06:49.118116075 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmtls/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/dcmtls/configure	2012-02-05 19:07:23.594783894 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/dcmwlm/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/dcmwlm/configure	2012-02-05 19:15:41.294800559 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/oflog/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/oflog/configure	2012-02-05 19:16:14.704801676 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/ofstd/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/ofstd/configure	2012-02-05 19:16:36.994802421 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
--- dcmtk-3.6.0/doxygen/configure.orig	2010-06-04 11:52:21.000000000 +0200
+++ dcmtk-3.6.0/doxygen/configure	2012-02-05 19:36:52.051509774 +0100
@@ -30,25 +30,25 @@
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		cd $thisdir
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	-c)
 		shift
 		cd "$configdir"
 		echo "running configure in config-directory"
-		./configure $*
+		./configure "$@"
 		;;
 	*)
 		echo "running configure for this module"
-		sh "$configdir/confmod" --srcdir=. $*
+		sh "$configdir/confmod" --srcdir=. "$@"
 		;;
 	esac
 else
 	echo "running configure for this module"
-	sh "$configdir/confmod" --srcdir=. $*
+	sh "$configdir/confmod" --srcdir=. "$@"
 fi	
 
 
